### PR TITLE
Changelog conversation update for 17.1

### PR DESCRIPTION
**Pull request by nickvergessen**
### 🖼️ Screenshots

grafik

### 🏁 Checklist

- [x] ⛑️ Tests (unit and/or integration) are included or not required
- [x] 📘 API documentation in `docs/` has been updated or is not required
- [x] 📗 User documentation in https://github.com/nextcloud/documentation/tree/master/user_manual/talk has been updated or is not required
- [x] 🔖 Capability is added or not needed

### CODE REVIEW

**Comment by nickvergessen** [comment]
Will summon the backport bot, but we should not merge it yet, so that it is translated for people when we update the beta.

File: lib/Chat/Changelog/Manager.php [hunk 1]
```diff
@@ -69,64 +69,68 @@ public function updateChangelog(string $userId): void {
 	public function getChangelogs(): array {
 		return [
 			$this->l->t(
-				"Welcome to Nextcloud Talk!\n"
+				"# Welcome to Nextcloud Talk!\n"
```
Comment (Antreesy) [inline lib/Chat/Changelog/Manager.php:72]: Not sure we should use `<h1>`, as per CEO / readers / someone else's standards it's required to use no more than 1 header on a web-page

Maybe stick with `##` for big headers, and `###` for subheaders?

Comment (nickvergessen) [reply lib/Chat/Changelog/Manager.php:72]:
```suggestion
				"## Welcome to Nextcloud Talk!\n"
```

**Review by Antreesy — APPROVED** [review]

**Pull request merged**